tests/foreman/ui_airgun/test_product.py

```diff
@@ -0,0 +1,40 @@
+"""Test class for Products UI
+
+:Requirement: Product
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from nailgun import entities
+
+from robottelo.datafactory import gen_string
+from robottelo.decorators import tier2
+
+
+@tier2
+def test_positive_create_in_different_orgs(session):
+    """Create Product with same name but in different organizations
+
+    :id: 469fc036-a48a-4c0a-9da9-33e73f903479
+
+    :expectedresults: Product is created successfully in both
+        organizations.
+
+    :CaseLevel: Integration
+    """
+    orgs = [entities.Organization().create() for _ in range(2)]
+    product_name = gen_string('alpha')
```

Comment:
Couple of suggestions, if you don't mind:
- how about parametrizing `product_name`? picking one random name as in original test sounds reasonable to me, we saw some issues with treating different utf8/hieroglyphs before.
- how about using 1 field to distinguish products, e.g. put organization name to product description and asserting proper description? That would make sure correct product was found, not the one from different organization.